Guide for users switching from LaTeX (#1103)
typst/typst: docs/src/general/guide-for-latex-users.md

>>>    ]),
>>>  )
>>>
>>>  par(justify: false)[
>>>    *Abstract* \
>>>    #abstract
>>>  ]
>>>
>>>  set align(left)
>>>  columns(2, doc)
>>>}
<<< #import "conf.typ": conf
#show: conf.with(
  title: [
    Towards Improved Modelling
  ],
  authors: (
    (
      name: "Theresa Tungsten",
      affiliation: "Artos Institute",
      email: "tung@artos.edu",
    ),
    (
      name: "Eugene Deklan",
      affiliation: "Honduras State",
      email: "e.deklan@hstate.hn",
    ),
  ),
  abstract: lorem(80),
)

Let's get started writing this article by
putting insightful paragraphs right here!
```

The [`{import}`]($scripting/#modules) statement makes
[functions]($type/function) from another file available. In this example, it
imports the `conf` function from the `conf.typ` file. The function can format
content as a conference article. We use the show rule to apply it to the
document and pass some metadata about the article to the `conf` function.
Finally, we can get started writing our article below!

<div class="info-box">

Functions are Typst's "commands" and can transform their
arguments to an output value, including document _content._ Functions can't

> generally manipulate anything they did not receive as an argument.

Reviewer: Multiple negations, I would simplify to: _Functions can generally manipulate only what they receive as an argument._

What does generally refer to? Are there exceptions? If there are, this would be a natural place to put a comment. Without followup this sentence seems unnecessary.

---
description: |
  Are you a LaTeX user? This guide explains the differences and
  similarities between Typst and LaTeX so you can get started quickly.
---

# Guide for LaTeX users
This page is a good starting point if you have used LaTeX before and want
to try Typst. We will explore the main differences between these two

> systems from a user perspective and you will learn how to use your LaTeX
> skills to get a head start with Typst.
> 
> <!-- Mention that Typst is not built upon LaTeX -->

Reviewer 2: ```suggestion
systems from a user perspective.
Although Typst is not built upon LaTeX and has a different syntax,
you will learn how to use your LaTeX skills to get a head start.
```

Just like LaTeX, Typst is a markup-based typesetting system: You compose your
document in a text file and mark it up with commands and other syntax. Then, you
use a compiler to typeset the source file into a PDF. However, Typst also
differs from LaTeX in several aspects: For one, Typst uses more dedicated syntax
(like you may know from Markdown) for common tasks. Typst's commands are also

> more principled: The all work the same, so unlike in LaTeX, you just to learn

Reviewer 3: ```suggestion
more principled: They all work the same, so unlike in LaTeX, you just to learn
```
